client/src/app/style.module.scss

```
@@ -25,5 +51,127 @@
             font-size: 18px;
         }
     }
+}
 
+.landing_section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 24px;
+    width: 100%;
+    text-align: center;
+}
+
+.landing_title {
+    font-weight: bold;
+    font-size: 48px;
```

```suggestion
    font-size: 48px;
    word-break: keep-all
```
word-break 추가 어때요..? 가독성 더 좋을 듯

---

오 좋은데요 바로 추가했습니다!